### PR TITLE
AISDK-123: Fix build warning by adding version tag

### PR DESCRIPTION
**Pull request by k-weng**
Fixes warning
```
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for ai.rev.speechtotext:revai-java-sdk-speechtotext:jar:1.1.0
[WARNING] 'build.plugins.plugin.version' for org.kohsuke:pgp-maven-plugin is missing. @ line 230, column 29
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
```

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -226,10 +226,10 @@
                             </execution>
                         </executions>
                     </plugin>
-
                     <plugin>
                         <groupId>org.kohsuke</groupId>
                         <artifactId>pgp-maven-plugin</artifactId>
+                        <version>1.1</version>
```
Comment (k-weng) [inline pom.xml:232]: I believe this is the addition needed to remove the warning.

Comment (amikofalvy) [reply pom.xml:232]: @Chanc3 what do you think of this change?

Comment (Chanc3) [reply pom.xml:232]: Yes lets update that

**Review by Chanc3 — APPROVED** [review]
:shipit:

**Pull request merged**